src/libnml/rcs/rcs_print.cc

```diff
@@ -47,7 +47,7 @@ long rcs_print_mode_flags = PRINT_RCS_ERRORS;
 FILE *rcs_print_file_stream = NULL;
 char rcs_print_file_name[80] = "rcs_out.txt";
 
-char last_error_bufs[4][100];
+char last_error_bufs[4][256];
```

Comment:
Perhaps 256 here should be replaced with error_buf_size?

Reply:
good catch, thanks.